packages/gatsby/src/schema/context.ts

```diff
@@ -0,0 +1,51 @@
+import { GraphQLSchema } from "graphql"
```

Comment:
I understand why you'd do it in this PR but it does make reviews more difficult because github fails to pick up on the rename and has no way to manually ask to compare two files. Please next time do TS conversions in a separate PR.